modules/orbidderBidAdapter.md

```diff
@@ -12,19 +12,46 @@ Module that connects to orbidder demand sources
 
 # Test Parameters
 ```
-var adUnits = [{
-    code: '/105091519/bidder_test',
-    mediaTypes: {
-        banner: {
-            sizes: [728, 90]
-        }
+var adUnits = [
+    {
+        code: 'test_banner',
+        mediaTypes: {
+            banner: {
+                sizes: [728, 90]
+            }
+        },
+        bids: [{
+            bidder: 'orbidder'
```

Comment:
Hi @arneschulz1984, missing `,` here, on line 49 and line 51, everything else looks good to me.

Reply:
Hi @aleksatr, thanks for the hint. Fixed to valid JavaScript with commit: https://github.com/prebid/Prebid.js/pull/7047/commits/e6e824403b3f150d85939ebed943a8c8abf8cb2e